api/src/main/java/org/openmrs/obs/handler/AbstractHandler.java

```diff
@@ -36,6 +43,12 @@ public class AbstractHandler {
 	
 	protected NumberFormat nf;
 	
+	@Autowired
```

Comment:
Is there a specific reason for keeping these access modifiers as default?

Reply:
Default lets us easily mock them in tests that we put in the same package. I included them in a constructor so it's not needed in this case, but sometimes constructor would be too long. I don't know if we have a convention around that, but it's something I find quite common in spring code.  They could be protected as well, but then mocking is only feasible via the constructor.